skipper.go

```diff
@@ -957,6 +961,33 @@ func (o *Options) KubernetesDataClientOptions() kubernetes.Options {
 	}
 }
 
+func (o *Options) OAuthGrantOptions() *auth.OAuthConfig {
+	oauthConfig := &auth.OAuthConfig{}
+
+	oauthConfig.AuthURL = o.OAuth2AuthURL
+	oauthConfig.TokenURL = o.OAuth2TokenURL
+	oauthConfig.RevokeTokenURL = o.OAuth2RevokeTokenURL
+	oauthConfig.TokeninfoURL = o.OAuthTokeninfoURL
+	oauthConfig.SecretFile = o.OAuth2SecretFile
+	oauthConfig.ClientID = o.OAuth2ClientID
+	oauthConfig.ClientSecret = o.OAuth2ClientSecret
+	oauthConfig.ClientIDFile = o.OAuth2ClientIDFile
+	oauthConfig.ClientSecretFile = o.OAuth2ClientSecretFile
+	oauthConfig.CallbackPath = o.OAuth2CallbackPath
+	oauthConfig.AuthURLParameters = o.OAuth2AuthURLParameters
+	oauthConfig.Secrets = o.SecretsRegistry
```

Comment:
```
oauthConfig.SecretsProvider = grantSecrets
```
missing?

Reply:
Okay, I see it below
https://github.com/zalando/skipper/pull/2953/files#diff-9b0b16d06f27b640eb77f6ba0dee84df6a660528e5cccd81839e0330c4b92dd2R1815